### DISCUSSION TITLE
openmdx 2.18.0 Tag

### DISCUSSION THREAD

**Discussion by brunoamacos** [openmdx/openmdx#68] [Q&A]
As i can see openCRX 5.2.1 is using openMDX 2.18.+. 
But  I dont find the 2.18. Tag in Github.

**Comment by DlRTY-HARRY** [comment]
The openMDX 2.18.0 tag is called [openmdx-v2.18.0](https://github.com/openmdx/openmdx/releases/tag/openmdx-v2.18.0)
